examples/message_checker.py

```diff
@@ -0,0 +1,12 @@
+import voltage
+
+client = voltage.Client()
+
+@client.listen('message')
+async def on_message(message):
+  ungodly_words = ["sus", "baka", "suppose", "real", "amogus"]
+  if message.content.lower() in ungodly_words:
```

Comment:
but why

Reply:
also as I've mentioned, this is quite valueless as an example, add a command that adds words that are censored using the commands framework and I'll consider it I suppose.

Reply:
yeah epic ok